scheduler/scheduler_test.go

```diff
@@ -0,0 +1,167 @@
+package scheduler
+
+import (
+	"testing"
+	"time"
+
+	"github.com/robfig/cron/v3"
+)
+
+const (
+	everySecond   = "* * * * * *"
+	maxRunTime    = 200 * time.Millisecond
+	scheduleLimit = 5 * time.Second
+)
+
+func TestScheduleTask(t *testing.T) {
+	s := NewWithSeconds()
+
+	// Add task to scheduler
+	jid, err := s.Add(everySecond, nil, maxRunTime, 0)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	var noEnt cron.Entry
```

Comment:
you could make this a global like zeroEntry :)

Reply:
Done